<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QListWidget" name="listWidget_condition">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>选股条件</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_ma">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="lineEdit_ma1"/>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>日均线与</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_ma2"/>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>日均线</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_maCondition"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_macd">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Diff线</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_macdCond"/>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Dea线</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>0轴</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_position"/>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>%</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_percent"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>选股周期：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_period"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>选股条件：</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QListWidget" name="listWidget_pickupConds">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pb_add">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>加入</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_delete">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_pickup">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>选股</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_specificDate">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>指定时间段</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>开始时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>结束时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEdit_2">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_add</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>on_addPickupCond()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>801</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_pickup</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>on_pickup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>660</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_delete</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>on_deletePickupCond()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>420</y>
    </hint>
    <hint type="destinationlabel">
     <x>806</x>
     <y>430</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_specificDate</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>on_specificDate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>759</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>807</x>
     <y>396</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_addPickupCond()</slot>
  <slot>on_deletePickupCond()</slot>
  <slot>on_pickup()</slot>
  <slot>on_specificDate()</slot>
 </slots>
</ui>
